packages/fxa-payments-server/src/styles/tailwind.css

```diff
@@ -37,3 +37,60 @@
 .payment-error .legal-heading {
   @apply mt-5;
 }
+
+/* shared styles used in PlanDetails and PlanUpgradeDetails */
+.btn {
+  @apply border-transparent text-blue-500 leading-5 my-2 outline-none py-2 pr-2 pl-6 relative focus:border focus:border-solid focus:border-blue-400 focus:py-2 focus:pr-4 focus:pl-10 focus:shadow-none;
+}
+
+.arrow {
+  @apply before:bg-[url('../components/PlanDetails/images/chevron.svg')] before:content-[''] before:absolute before:h-4 before:w-6 before:top-2.5  before:left-0 focus:before:left-3.5;
+}
+
+.coupon-info {
+  @apply flex items-center justify-center gap-2 pt-6 text-sm text-green-700;
+}
+
+.label {
+  @apply font-semibold text-sm;
+}
```

Comment:
I know the old style was 15px but can we possibly bump this up to 16px (`text-base`) instead of dropping it down to 14px (`text-sm`)? The label looks a bit small next to the price.

Nit: can we consider changing the class name to be a tad more specific like "price-label" or "label-price" or "label-total", etc? It's a little too ambiguous if it's only applied to this particular label. Unless we plan on all labels defaulting to this style?

Note: the colour has also changed and now matches with the price. I am aligned with this change as it improves contrast, UX/a11y in how the user visually connects the label to the price (i.e., easier to skim or spot at a glance), and maintains consistency with the other labels and headers. 👍

Reply:
I had changed the color and size to match the other "labels" on the greater page, but I'm aligned with bumping "Total" to `text-base`.

I'll change `label` to `label-total`. I think, at a later time, we could possibly make a generalized shared style for similar "labels" though that could help us out - I listed a few examples in components below.

- "Product name" and "Product details" in `PlanDetails`,
- "Name as it appears on your card" and "Your card" in `PaymentForm`,
- "Order Details" and "Payment Information" in `PaymentConfirmation`